dashboard/app/controllers/plc/user_course_enrollments_controller.rb

```diff
@@ -3,13 +3,19 @@ class Plc::UserCourseEnrollmentsController < ApplicationController
   authorize_resource only: :create
 
   def index
-    @user_course_enrollments = @user_course_enrollments.where(user: current_user)
+    @user_course_enrollments = @user_course_enrollments.where(user: current_user) if @user_course_enrollments
```

Comment:
Yeah - this bothered me. My interpretation of principle-of-least-surprise would be that it would return empty array (or the ActiveRecord equivalent)